frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -77,6 +82,32 @@ const typeMap = {
             ).url
         },
     },
+    RetentionTable: {
+        className: 'retention',
+        element: RetentionTable,
+        icon: QuestionCircleOutlined,
```

Comment:
Not sure if there better options, but these question marks aren't very illustrative

Reply:
oh good catch i meant those to be placeholders. Yeah, definitely need suggestinos with how to mount a default size. didn't realize it would lock the sizing. Let me take a look again

Reply:
I fixed the width resizing for paths. But for retention and paths, I'm not sure how to fix the resizing height issue

Reply:
I think I have a solution: below a certain size, use
```css
transform-origin: left top;
transform: scale(0.6)
```
And automatically scale based on the resizing on the table itself. It's a bit more work so I'd accept the answer of "I'm not going to bother"